### DISCUSSION TITLE
Skeleton

### DISCUSSION THREAD

**Discussion by Plastisheep** [Draylar/identity#319] [Ideas]
I keep writing these as I play. Sorry lol. I feel like the skeleton should automatically have an infinity-like effect on bows and/or more powerful bows (maybe a slight bit of aimbot or something too?). Just throwing ideas out there!

**Comment by James103** [comment]
The effect should be more like Infinite Quiver instead of Infinity itself. In other words, an X% chance to not consume an arrow when shooting one. By default, P(arrow is not consumed) = 50% (subject to change), but this would increase to 100% with the Infinity enchantment.
